### DISCUSSION TITLE
Oops! No text generated from Ollama, Please try again.

### DISCUSSION THREAD

**Discussion by wwjCMP** [open-webui/open-webui#2830] [General]
# Bug Report

When using a model that supports a long context of 256K for dialogue with PDFs, this error occurs, but clicking 'Continue' allows for the generation of an answer.

**Comment by wwjCMP** [comment]
Should the response time be increased to adapt to long context scenarios?

**Comment by luffy9x** [comment]
I have the same issuse, is there anyone have the solution ?

**Comment by mark-kazakov** [comment]
I have the same issue when running the image `llama3.2-vision:latest`

ollama version: `0.4.2`

openwebui version: `v0.3.35`

**Comment by fer000n** [comment]
in model "granite-3.1-3b" got this error: Oops! No text generated from Ollama, Please try again.
